File: src/main/java/com/IconPippi/simtriggers/triggers/ConnectionCloseTrigger.java
package com.IconPippi.simtriggers.triggers;

public class ConnectionCloseTrigger extends Trigger {

	protected ConnectionCloseTrigger(String method) {
		super(method);
	}

	@Override
	protected void trigger() {
		this.getScriptLoader().invokeFunction(method, (Object[]) null);
	}

}

File: src/main/kotlin/com/simtriggers/fsx/triggers/ConnectionCloseTrigger.kt
package com.simtriggers.fsx.triggers

/**
 * 05/08/2019
 * Called once the connection is closed
 *
 * @author IconPippi
 */
class ConnectionCloseTrigger(private val function: String) : Trigger(function, null) {

    override fun trigger(eventName: String?) {
        scriptLoader.invokeFunction(function, null)
    }

}